lmod_proxy/edx_grades/actions.py

```diff
@@ -119,3 +129,13 @@ def get_sections(gradebook, form):
         data,
         not bool(error_message)
     )
+
+
+def _get_assignment_name_max_points(form):
+    try:
+        csv_data = form.datafile.data.stream.read()
+        short_name = csv_data[0][1]
+        max_points = csv_data[1][2]
+        return short_name, max_points
+    except IOError:
+        raise
```

Comment:
Since you're just reraising this exception you should be able to remove the try and except completely